.stylelintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+  "rules": {
+    "block-closing-brace-newline-after": "always",
+    "block-no-empty": true,
+    "block-opening-brace-space-before": "always",
+    "color-hex-case": "lower",
+    "color-hex-length": "short",
+    "color-named": "never",
+    "color-no-invalid-hex": true,
+    "declaration-bang-space-before": "always",
+    "declaration-block-no-duplicate-properties": true,
+    "declaration-block-semicolon-newline-after": "always",
+    "declaration-colon-space-after": "always",
+    "declaration-block-trailing-semicolon": "always",
+    "function-comma-space-after": "always",
+    "function-url-quotes": "always",
+    "indentation": [4, { "severity": "warning" }],
+    "length-zero-no-unit": true,
+    "max-nesting-depth": 5,
+    "no-missing-end-of-source-newline": true,
+    "no-eol-whitespace": true,
+    "rule-empty-line-before": "always",
+    "string-quotes": "single"
```

Comment:
For each of those rules, could you detail what `scss_lint` rule they replace? Not in the code, but just in this PR so we know how close of a mapping there is between the old and new config. I'd also be interested to know what `stylelint` rules exist that have no equivalent in `scss_lint` and we could start leveraging, although that's beyond the scope of this one PR.